packages/colorpickers/src/elements/ColorPicker/index.tsx

```diff
@@ -0,0 +1,284 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, {
+  useEffect,
+  useCallback,
+  useContext,
+  useReducer,
+  forwardRef,
+  HTMLAttributes,
+  useMemo,
+  useRef
+} from 'react';
+import PropTypes from 'prop-types';
+import { ThemeContext } from 'styled-components';
+import { Label } from '@zendeskgarden/react-forms';
+import { ColorWell } from './ColorWell';
+import {
+  StyledHue,
+  StyledSliderGroup,
+  StyledLabel,
+  StyledInput,
+  StyledAlpha,
+  StyledSliders,
+  StyledHexField,
+  StyledRGBAField,
+  StyledInputGroup,
+  StyledPreview,
+  StyledColorPicker,
+  StyledCheckered,
+  StyledAlphaGradient,
+  StyledAlphaField,
+  StyledHueField
+} from '../../styled';
+import { areColorsEqual, convertStringToColor, getInitialState, reducer } from './reducer';
+import { IColor, IHSVColor } from '../../utils/types';
+
+export interface IColorPickerLabels {
+  hueSlider?: string;
+  alphaSlider?: string;
+  hex?: string;
+  red?: string;
+  green?: string;
+  blue?: string;
+  alpha?: string;
+}
+
+export interface IColorPickerProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, 'color' | 'onChange'> {
+  /** A default color for a uncontrolled color picker. Can be a valid color string or IColor state */
+  defaultColor?: string | IColor;
+  /** A color value for a controlled color picker. Can be a valid color string or IColor state */
```

Comment:
*Sets the color for a controlled color picker

~. Can be a valid color string or IColor state~ is unnecessary as the info is apparent from the prop sheet data.